app/src/main-process/main.ts

```diff
@@ -104,6 +105,15 @@ if (__WIN32__ && process.argv.length > 1) {
   }
 }
 
+if (__DARWIN__) {
```

Comment:
I like this check here because `if (false) {` should be dead code for non-macOS platforms, but I also like the `shellNeedsPatching` check in case someone tries to call it from elsewhere in the future. 🤔